packages/jobs/src/job-definitions/events/publishToAnalyticsJob.ts

```diff
@@ -12,28 +13,24 @@ export const publishToAnalyticsJob = async (job: Job<LatitudeEvent>) => {
     workspaceId = event.data.workspaceId
   }
 
-  const client = PostHogClient()
+  if (!userEmail) return
+
+  if (env.NODE_ENV !== 'production') {
+    console.log('Analytics event captured:', event.type)
```

Comment:
Do we need this console.log

Reply:
It's useful for local development tbh

Reply:
But not when not developing events. I would add when working on it. The rest of the time is just noise

Reply:
ok will remove then

Reply:
Whe have this 
https://github.com/latitude-dev/latitude-llm/blob/main/packages/core/src/lib/debug.ts

If you use `debug` this console logs will only appear when you run 

```json
NODE_DEBUG=latitude:debug next dev --turbo
```

Which now is always but we can move this to `pnpm dev:debug`

All done easy just use `debug` instead of console.log

Reply:
This is the standard node way of doing console.logs